### ISSUE TITLE
Key request

### ISSUE THREAD

**Issue by OvermindDL1** [vguillou/swipe-action#4]
To make more efficient updating, can a 'key' attribute or so be added to this, just whenever it changes from its previous value then reset the state of this?  This would allow it to be used within dom-repeat when the items change on occasion (like removing them on delete), where currently if any are in a swiped state then they are still swiped even as the items are re-rendered with new content.

Basically just adding a new 'key' or 'keyed' or so property, have an observer on it, whenever it changes and does not equal its previous value then call `this.reset(false)` or so (perhaps with another property for whether it should be false/true?).

**Comment by OvermindDL1** [comment]
To be specific, can work around it by iterating over the elements in a dom-repeat after it fills them (though can be slow), or can also use a calculated property based on some item key to set activatedActionId back to undefined, though both feel quite hacky to work around a lazy dom, a key would be setting a proper observable on the other hand
